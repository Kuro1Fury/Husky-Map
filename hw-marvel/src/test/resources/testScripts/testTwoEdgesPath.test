# Tests that the path between two edges in the graph can be found

CreateGraph A

AddNode A n1
AddNode A n2
AddNode A n3
AddEdge A n1 n2 1to2
AddEdge A n2 n3 2to3

FindPath A n1 n3

FindPath A n3 n1

