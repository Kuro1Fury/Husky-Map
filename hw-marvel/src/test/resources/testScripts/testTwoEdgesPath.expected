# Tests that the path between two edges in the graph can be found

created graph A

added node n1 to A
added node n2 to A
added node n3 to A
added edge 1to2 from n1 to n2 in A
added edge 2to3 from n2 to n3 in A

path from n1 to n3:
n1 to n2 via 1to2
n2 to n3 via 2to3

path from n3 to n1:
no path found